pm/ticket.go

```diff
@@ -174,15 +176,17 @@ func (t *Ticket) Hash() ethcommon.Hash {
 // [32..63] = CreationRoundBlockHash
 // See: https://github.com/livepeer/protocol/blob/pm/contracts/pm/mixins/MixinTicketProcessor.sol#L94
 func (t *Ticket) AuxData() []byte {
-	if t.CreationRound == 0 && (t.CreationRoundBlockHash == ethcommon.Hash{}) {
-		// Return empty byte array if both values are 0
-		return []byte{}
-	}
+	return (&TicketExpirationParams{
+		CreationRound:          t.CreationRound,
+		CreationRoundBlockHash: t.CreationRoundBlockHash,
+	}).AuxData()
+}
 
-	return append(
-		ethcommon.LeftPadBytes(big.NewInt(t.CreationRound).Bytes(), uint256Size),
-		t.CreationRoundBlockHash.Bytes()...,
-	)
+func (t *Ticket) ticketExpirationParams() *TicketExpirationParams {
```

Comment:
Since the method receiver makes it clear that we are referring to a ticket what do you think about shortening this method name to `expirationParams()`?

Reply:
fixed in 87762826